docs/whats_new/version_ongoing.rst

```diff
@@ -28,3 +28,50 @@ There is not API changes, therefor, as user, your tests should not be affected.
 Agnostic CCSocketCan
 ^^^^^^^^^^^^^^^^^^^^
 Incompatibilities with the agnostic proxy are now resolved. You should be able to use it again.
+
+Proxy Auxiliary adaption
```

Comment:
Merging conflicts where not fixed correctly. All this section was reworked and put at l.18 to 30